src/config/dev-dependencies.ts

```diff
@@ -0,0 +1,17 @@
+import RegistrationService from "../application/registration/Service";
+import RegistrationInMemoryGateway from "../infrastructure/gateway/RegistrationInMemoryGateway";
+import RegistrationController from "../presentation/controller/registration/RegistrationController";
+
+const registrationGateway: RegistrationInMemoryGateway =
+  new RegistrationInMemoryGateway();
+const registrationService: RegistrationService = new RegistrationService(
+  registrationGateway
+);
+const registrationController: RegistrationController =
+  new RegistrationController(registrationService);
+
+export const appDevDependencies = {
+  registrationGateway,
```

Comment:
I don't think the gateway and service need to be supplied here anymore.

Reply:
Unit tests use the service and check the gateway 
So we need them for dev dependencies, not for real dependencies

Reply:
Looks like this isn't the case, as we saw in the demo.

Reply:
because in my demo, I recreated the dependencies instead of retrieving them from here (the stress of the live)